### ISSUE TITLE
Implement "explanation chains"

### ISSUE THREAD

**Issue by nwawrzyniak** [feildmaster/UnderScript#31]
Sometimes you don't get information about generated cards (green rarity) easily. Especially for them it should be possible to check the keywords of the card on another card.

Example: C-Round

If you're in [Crafting](https://undercards.net/Crafting), search for ```C-Round``` and hover over the word ```Crown``` in his text you obviously get a preview of the card ```Crown```. In the ```Crown``` card text there is the term ```K-Round```, but you cannot hover over that term to get a preview, since the ```Crown``` card vanishes as soon as you move the cursor away from its name on ```C-Round```.

TL;DR it should be possible to make a card that you see because you hover over another card stay on screen and be checkable.

**Comment by feildmaster** [comment]
I could do that ... but then I'd have to completely override the function onu uses to display cards. Not really a problem but maybe(?) too much to contend with. I'll look into it.
